packages/ngx-table/src/lib/components/table-cell/table-cell.component.ts

```diff
@@ -39,6 +42,10 @@ export class TableCellComponent implements OnChanges {
     return !!this.component;
   }
 
+  public hasColumnClass() {
```

Comment:
There is no need for this method, you can just use `!!this.columnClass` in the template